src/handlers/swap_handlers.ts

```diff
@@ -233,6 +271,30 @@ export class SwapHandlers {
                 .getPriceComparisonFromQuote(CHAIN_ID, marketSide, quote)
                 ?.map((sc) => priceComparisonUtils.renameNative(sc));
         }
+
+        if (quote.extendedQuoteReport && kafkaProducer) {
+            const bytesPos = quote.data.indexOf('869584cd');
```

Comment:
hmm, what is this magic value? Can we add a comment regarding where this comes from?